test/js/utils/misc.js

```diff
@@ -23,4 +23,4 @@ export const spawnPrinter = (color, stdout, stderrorPrefix) => process => {
   });
 }
 
-export const SHELL_COLOR_BLUE = "\x1b[34m"
+export const SHELL_COLOR_BLUE = "\x1b[32m"
```

Comment:
This export constant name doesn't match up with the import name in `main.js` file above (`SHELL_COLOR_GREEN`)

Reply:
Didn't save the file before pushing 🤦‍♂ done now.